Expensify/expensify-common
# Fix Broken Refs in Combobox JS

@chiragsalian will you please review this?

Seems like combobox at one point at this feature? I noticed the refs on the text input were set as `ref={ref = this.value = ref}` and then used in the component as `this.refs.value`. I believe this is the pattern for string refs and that's why the component was not highlighting the text whenever `$.focus().select()` was called. I updated references to this.value to reflect the way the ref has been set, but for some reason I don't understand this caused some errors in the attendees selector whenever `this.combobox.value` was called. Threw everything into a setTimeout so the `this.combobox` would be accessible from the AttendeesSelector and that seemed to work. Everything should be working but I would first merge the changes to AttendeesSelector in Web-Expensify before merging these changes.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/92367

# Tests / QA
1. Go to any combobox in expenses with a value set
1. Click on it and the text should be highlighted.
